controller.go

```diff
@@ -307,6 +312,30 @@ func getPDBs(labels map[string]string, pdbs []pv1beta1.PodDisruptionBudget, sele
 	return matchedPDBs
 }
 
+func getOwnedPDBs(pdbs []pv1beta1.PodDisruptionBudget, owner kubeResource) []pv1beta1.PodDisruptionBudget {
+	ownedPDBs := make([]pv1beta1.PodDisruptionBudget, 0, len(pdbs))
+	for _, pdb := range pdbs {
+		if isOwnedReference(owner, pdb.ObjectMeta) {
+			ownedPDBs = append(ownedPDBs, pdb)
+		}
+	}
+	return ownedPDBs
+}
+
+// isOwnedReference returns true if the dependent object is owned by the owner
+// object.
+func isOwnedReference(owner kubeResource, dependent metav1.ObjectMeta) bool {
+	for _, ref := range dependent.OwnerReferences {
+		if ref.APIVersion == owner.APIVersion() &&
```

Comment:
Wasn't there a bug where these don't get populated?

Reply:
This was/is the case for informers: https://github.com/kubernetes/client-go/issues/308 We just do normal lists for now.